tox.ini

```diff
@@ -21,10 +21,10 @@ pypi_filter = https://raw.githubusercontent.com/astropy/ci-helpers/main/pip_pinn
 # Pass through the following environment variables which are needed for the CI
 passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI IS_CRON ARCH_ON_CI
 
-# For coverage, we need to pass extra options to the C compiler
 setenv =
+    # For coverage, we need to pass extra options to the C compiler
     cov: CFLAGS = --coverage -fno-inline-functions -O0
-    image: MPLFLAGS = --mpl
+    image: MPLFLAGS = -m "mpl_image_compare" --mpl --mpl-generate-summary=html --mpl-results-path={toxinidir}/results --mpl-hash-library={toxinidir}/astropy/tests/figures/{envname}.json --mpl-baseline-path=http://data.astropy.org/testing/astropy/figures/{envname}/ --remote-data=astropy -W ignore::DeprecationWarning
```

Comment:
Why do we have the old data repo here and not the new one? If there aren't any figures to compare it against, it will still work.

Reply:
Oops, mistake